cardano-testnet/test/Test/Cli/Alonzo/LeadershipSchedule.hs

```diff
@@ -499,15 +498,17 @@ hprop_leadershipSchedule = H.integration . H.runFinallies . H.workspace "alonzo"
 
   expectedLeadershipSlotNumbers <- H.noteShowM $ fmap (fmap slotNumber) $ H.leftFail $ J.parseEither (J.parseJSON @[LeadershipSlot]) scheduleJson
 
+  maxSlotExpected <- H.noteShow $ maximum expectedLeadershipSlotNumbers
+
   H.assert $ not (L.null expectedLeadershipSlotNumbers)
 
   leadershipDeadline <- H.noteShowM $ DTC.addUTCTime 90 <$> H.noteShowIO DTC.getCurrentTime
 
-  H.assertByDeadlineMCustom "Leader schedule is correct" leadershipDeadline $ do
+  -- Leader schedule is correct
+  H.byDeadlineM 10 leadershipDeadline $ do
```

Comment:
`"Condition not met by deadline"` is a somewhat opaque error. Can we implement a custom `byDeadlineM` to maintain the custom error messages used by `assertByDeadlineMCustom`

Reply:
Will require this change: https://github.com/input-output-hk/hedgehog-extras/pull/16

Reply:
Updated to include better assertion messages.